core/templates/core/panel.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Panel Gwardii{% endblock %}
+
+{% load list_events %}
+
+{% block content %}
+    <div class="flex flex-row nowrap flex-grow h-full overflow-auto">
+        <div id="sidebar" class="hidden fixed top-0 md:relative md:block min-w-72 max-w-72 h-full bg-gray-800 shadow-2xl overflow-auto z-[0]">
+        <div class="flex flex-col gap-y-3 pb-28 md:pb-0" id="sidebar-content">
+            <a href="/" class="mx-4 py-3 my-2 h-12 max-w-full text-lg font-bold hover:underline">Strona Główna</a>
+            {% for namespace, links in sidebar_links.items %}
+            <div class="flex flex-grow flex-col divide-y divide-y-reverse divide-gray-600">
+                <h3 class="mx-4 py-3 my-2 h-12 max-w-full text-lg font-bold">{{ namespace }}</h3>
```

Comment:
Why to manually define height here? And for example `max-w-full` doesn't change anything afaik...